### ISSUE TITLE
Icons is cut in them's upper and lower parts

### ISSUE THREAD

**Issue by D0ve** [OpenLauncherTeam/openlauncher#402]
When moving smartphone from vertical to horizontal position   icons has cut (resection) in them's upper and lower parts.

**Comment by serebryakov-av** [comment]
Think it could be better if rotate only icons (and may be widgets) instead of whole screen.

**Comment by DI555** [comment]
Bug still exist in v6.0 .(
I'd love to rotate whole screen).

But, never the less, rotate only icons (and/or) widgets  would be great optional 2nd option in settings !!! And would have amazing looking) !!!

**Comment by dkanada** [comment]
Yeah, I didn't do any work on this because I couldn't think of a decent solution. Only rotating icons would break the paging because you would have to swipe vertically to switch pages. A proper solution for this would require a lot of changes to the code. Probably a new AppItemView class that scaled based on the parent container size and a new SmoothViewPager for the horizontal view that had a lot of code to properly set the row and column count.
